sim/warrior/items.go

```diff
@@ -277,6 +277,8 @@ func (warrior *Warrior) ApplyRegicideWarriorEffect(itemID int32, aura *core.Aura
 			Duration:  time.Second * 15,
 		})
 	})
+	// Regicide aura for APL use on trigger from stacks
+	regicideAura := warrior.NewTemporaryStatsAura("Regicide", core.ActionID{SpellID: 1231436}, stats.Stats{}, 10*time.Second)
```

Comment:
Since this isn't an aura that affects stats we can just do a simple `RegisterAura` instead of the empty stats like thid